chain/network/src/network_protocol.rs

```diff
@@ -0,0 +1,424 @@
+/// Contains type that belong to the `network protocol.
+///
+/// We need to maintain backwards compatibility, all changes to this file needs to be reviews.
```

Comment:
This warning should be a LOT larger - and more scary...

WARNING ACHTUNG WARNING
WARNING ACHTUNG WARNING

Changing anything in this file might break our network protocol - please think about backwards compatibility and make sure to loop in someone from the core/network team in your PR.